tests/test_spherePoint.py

```diff
@@ -92,26 +92,37 @@ def positions():
         ]
         return points
 
-    def testInit2ArgErrors(self):
-        """Test if 2-argument form of __init__ handles invalid input.
+    def testLongLatConstructors(self):
+        """Test that the longitude, latitude constructors handle invalid input.
         """
         # Latitude should be checked for out-of-range.
         for lat in self._poleLatitudes:
             with self.assertRaises(pexEx.InvalidParameterError):
                 SpherePoint(0.0*degrees, self.nextUp(lat))
             with self.assertRaises(pexEx.InvalidParameterError):
+                SpherePoint(0.0, self.nextUp(lat).asDegrees(), degrees)
```

Comment:
Might be cleaner to call `np.nextafter` directly, or modify `nextUp` and `nextDown` to work on floating-point numbers.

Reply:
The current code is a bit clumsy but is obviously the same test, which I think is worth the clumsiness. I like the fact that nextUp works on Angles and am reluctant to change that.

Reply:
I meant "in addition to Angles", but ok.